webapp/components/topbar/Topbar.js

```diff
@@ -187,6 +187,7 @@ class Topbar extends Component {
             <LaunchDialog
               open={dialogOpen}
               onRequestClose={() => this.handleClose()}
+              experimentInDesign={this.props.experimentInDesign}
```

Comment:
There is an edge case when we click on `Simulate` when the experimentInDesign moves to a different state but the LaunchDialog is still rendered, this throws an Error. In that case experimentInDesign is undefined.

```
Uncaught TypeError: Cannot read property 'name' of undefined
    at LaunchDialog (LaunchDialog.js:270)
    at renderWithHooks (react-dom.development.js:14803)
    at updateFunctionComponent (react-dom.development.js:17034)
    at beginWork (react-dom.development.js:18610)
    at HTMLUnknownElement.callCallback (react-dom.development.js:188)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:237)
    at invokeGuardedCallback (react-dom.development.js:292)
    at beginWork$1 (react-dom.development.js:23203)
    at performUnitOfWork (react-dom.development.js:22154)
    at workLoopSync (react-dom.development.js:22130)
```

I would suggest to pass `experimentName` and `numberOfTrials` as props to the LaunchDialog and provide default values  (empty string) in case experimentInDesign is undefined.